UserTools/LikelihoodFitterCheck/LikelihoodFitterCheck.cpp

```diff
@@ -101,12 +137,13 @@ bool LikelihoodFitterCheck::Execute(){
   double dx = dl * trueDirX;
   double dy = dl * trueDirY;
   double dz = dl * trueDirZ;
-  int nbins = 200;
-  double dlpara[200], dlfom[200];
-  for(int j=0;j<200;j++) {
-    seedX = trueVtxX - 50*dx + j*dx;
-    seedY = trueVtxY - 50*dy + j*dy;
-    seedZ = trueVtxZ - 50*dz + j*dz;
+  int nbins = 400;
+  double dlpara[400], dlfom[400];
```

Comment:
is there any reason to accumulate these in an array? It seems like `dlfom[j]` is set to `fom`, and then the only time it's used is in the next line to access `dlfom[j]`, where `fom` is still in scope. Just use `fom`? Same for `dlpara`.